codecparsers/vp9quant.h

```diff
@@ -22,9 +22,9 @@ extern "C" {
 
 int clamp(int value, int low, int high);
 
-int16_t vp9_dc_quant(int qindex, int delta);
+int16_t vp9_dc_quant(int bit_depth, int qindex, int delta);
```

Comment:
better change protype  to  vp9_dc_quant(VP9_BIT_DEPTH bit_depth, int qindex, int delta);